src/use-connect.js

```diff
@@ -35,9 +35,9 @@ export const useUtilityConnect = () => {
   });
 
   const pollForFactory = useCallback(() => {
-    const { _ArcadiaUtilityConnect } = window;
-    if (_ArcadiaUtilityConnect) {
-      setFactory(_ArcadiaUtilityConnect);
+    const { _ArcConnect } = window;
+    if (_ArcConnect) {
+      setFactory(_ArcConnect);
```

Comment:
need to change this here: https://github.com/ArcadiaPower/utility-connect/blob/main/src/index.js

Reply:
DONE